storage/sectorblocks/sectorblocks.go

```diff
@@ -48,7 +48,7 @@ func DsKeyToDealID(key datastore.Key) (uint64, error) {
 }
 
 type SectorBuilder interface {
```

Comment:
Actually I think we are not importing this interface, but the one from Lotus.

Reply:
You're right, I've removed the entire file as its not being used.